bucket_restart/output_active

```diff
@@ -0,0 +1 @@
+output_0002
```

Comment:
This is an output file that we don't track in the ClimaLand repo, so we should remove it from here